lit-base/lambda_function.py

```diff
@@ -0,0 +1,24 @@
+from model_service import preprocess, inference, postprocess
+
+
+class BaseResponse:
```

Comment:
1. move to a another file
2. Make it inherit a dict - 

class BaseResponse(dict): 
 ....

this way a dict will be returned

Reply:
Different from Java, the correct way to organize Python file is catering readability instead of enforcing one class per file. 

Actually after considering carefully, I'm not sure about the idea of creating a class now. Let's wait a bit for more inputs.

Reply:
I went over this with MAR folks, and to support MAR we will prefer a BaseResponse. However, inherit from dict should be reconsidered carefully for two reasons:
1. HTTP headers is not a map, it's a list of key value pair. We need a way to format it into dictionary so that Lambda can consume. 
2. http://neethack.com/2017/04/Why-inheritance-is-bad/

Reply:
@kalyc `BaseResponse` is not `dict`,  You should use Inheritance when your classes can fit `is a` paradigm ie., The idea behind it is `old code using new code`. If a user has written code that uses a base class, he can just pass the new improved Derived class without much code change.

Reply:
@nswamy `BaseResponse` is structured as a dictionary - there are key value pairs associated with the status code, headers, body. With the current implementation, a method explicitly needs to be added `get_dict` and this route can be avoided. Hence the suggestion above.